tsconfig.json

```diff
@@ -40,7 +40,8 @@
         "**/public",
         "**/protobuf-patches",
         "**/trezor-common",
-        "**/jest.config.js"
+        "**/jest.config.js",
+        "**/suite-desktop/scripts"
```

Comment:
not sure about this @Nodonisko 
Without it, there is a problem in `build.ts` on line 9 `import { suiteVersion } from '../../suite/package.json';`

However, then it ignores tsconfig for whole folder

Reply:
What about just ts-expect-error comment? Or ignore just that specific file not whole folder.

Reply:
ts-expect-error does not help

Reply:
Helped, thank you